src/main/java/seedu/mtracker/model/Instrument.java

```diff
@@ -32,13 +48,15 @@ public String getSentiment() {
         return sentiment;
     }
 
-    public abstract String toString();
+    public String toString() {
+        return TextUi.createBoxDisplay(getStatusIcon());
+    }

```

Comment:
toString shouldnt return this, its not what the method is for, i think it shld be defined under another function